src/hooks/useLottery.ts

```diff
@@ -0,0 +1,82 @@
+import axios, { AxiosResponse } from "axios";
+import { useEffect, useState } from "react";
+import { Option } from "../components/atoms/Dropdown/Dropdown";
+import { LOTTERY_API, LOTTERY_DRAW_API, LOTTERY_RESULTS_API } from "../config/env";
+
+type Result = {
+  options: Option[];
+  getResult: (lotteryId: string) => Promise<DrawingResult>;
+};
+
+type Lottery = {
+  id: number;
+  name: string;
+};
+
+type DrawingResult = {
+  numbers: string[];
+};
+
+const request = <T>(endpoint: string): Promise<AxiosResponse<T>> => axios.get(endpoint);
+
+const useLottery = (): Result => {
+  const [options, setOptions] = useState<Option[]>([]);
```

Comment:
Instead of using `Option` here let's create a generic type. Also, let's rename options to lotteries.